tests/cases/arithmetic/shift_right_unsigned.test

```diff
@@ -0,0 +1,10 @@
+### SUBSTRAIT_SCALAR_TEST: v1.0
+### SUBSTRAIT_INCLUDE: '/extensions/functions_arithmetic.yaml'
+
+# basic: Basic examples without any special cases
+shift_right_unsigned(2::i32, 1::i32) = 1::i32
+shift_right_unsigned(1024::i32, 8::i32) = 4::i32
+shift_right_unsigned(301989888::i64, 3::i32) = 37748736::i64
```

Comment:
Should there be a test case that is different between shift right and shift right unsigned?